src/documentation/pages/components/forms/select/examples/_floating-labels.vue

```diff
@@ -0,0 +1,90 @@
+<template lang="pug">
+  <ComponentExample title="Floating labels" id="floating_labels" :tabs="exampleTabs">
+    p.-text(slot="example-description") 
+      | Floating labels are a solution to keep the placeholder visible when no label is attached to the select. 
+      | Chi only supports floating labels on <code>-lg</code> and <code>-xl</code> selects.
```

Comment:
Portal has a different description for Floating labels
<img width="550" alt="Floating-label" src="https://user-images.githubusercontent.com/92325462/149418605-964bb067-56f3-443d-84e1-d3b0fd78904c.png">
.

Reply:
i still see the same description for both themes 🤔